docs/network/operators.md

```diff
@@ -5,18 +5,18 @@ title: Network operators
 Etherlink relies on operators who run nodes.
 For information on the roles of these different nodes, see [Etherlink architecture](/network/architecture).
 
-## Nodes
+## Smart Rollup node operators
 
-Anyone can run Etherlink [EVM nodes](/network/evm-nodes) and [Smart Rollup nodes](/network/smart-rollup-nodes), and due to the optimistic nature of Tezos [Smart Rollups](https://docs.tezos.com/architecture/smart-rollups) it takes only one honest Smart Rollup node operator to keep Etherlink secure.
+Anyone can run Etherlink [Smart Rollup nodes](/network/smart-rollup-nodes), and due to the optimistic nature of Tezos [Smart Rollups](https://docs.tezos.com/architecture/smart-rollups) it takes only one honest Smart Rollup node operator to keep Etherlink secure.
 Honest Smart Rollup nodes can catch any misbehavior by other nodes by refuting their incorrect commitments.
 
-These organizations currently run Etherlink nodes:
+These organizations currently run Etherlink Smart Rollup nodes:
```

Comment:
```suggestion
These organizations currently run Smart Rollup nodes to defend Etherlink:
```

Reply:
That's not how we're using the word "defend" in the docs. How about this?
> These organizations currently run Etherlink Smart Rollup nodes in operator mode to post and defend commitments for the current state of Etherlink